requirements.txt

```diff
@@ -5,7 +5,7 @@ psycopg2==2.6.1
 sqlparse==0.1.19
 networkx==1.11
 csvkit==0.9.1
-paramiko==2.0.1
-sshtunnel==0.0.8.2
 snowplow-tracker==0.7.2
 celery==3.1.23
```

Comment:
`celery` is a dependency of `snowplow-tracker` -- do we still need to include it? unsure how it got in the requirements file